static/css/dropdown.less

```diff
@@ -253,3 +265,8 @@
   }
 }
 
+@media (max-width: 400px) {
+  .open .mm-dropdown-menu {
+    top: 101px;
+  }
+}
```

Comment:
This is unrelated but fixes a bug on mobile where the menu got disconnected from the button.